SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -281,10 +289,19 @@ extension PersonController: UITableViewDataSource {
     }
 }
 
-// MARK: - CollectionViewCellDelegate Methods
-extension PersonController: CollectionViewCellDelegate {
+// MARK: - PersonCollectionViewCellDelegate
+
+extension PersonController: PersonCollectionViewCellDelegate {
+    
+    func didTapNewsItem(_ news: News) {
+        if let url = URL(string: news.url) {
+            let vc = SFSafariViewController(url: url)
+            vc.modalPresentationStyle = .overFullScreen
+            self.present(vc, animated: true)
+        }
+    }
     
-    func collectionView(collectionviewcell: UICollectionViewCell?, index: Int, didTappedInTableViewCell: UITableViewCell) {
-        print("\(index) tapped")
+    func didTapMediaItem(_ media: Media) {
+        print("Did tap media")
```

Comment:
Was there anything in the design or ux for this? Could just be a modal VC with the media and a close button?